packages/amplify-cli-core/src/overrides-manager/override-skeleton-generator.ts

```diff
@@ -52,7 +62,10 @@ export async function buildOverrideDir(cwd: string, destDirPath: string): Promis
   const packageManager = getPackageManager(cwd);
 
   if (packageManager === null) {
-    throw new Error('No package manager found. Please install npm or yarn to compile overrides for this project.');
+    throw new AmplifyError('MissingOverridesInstallationRequirementsError', {
+      message: 'No package manager found.',
+      resolution: 'Please install npm or yarn to compile overrides for this project.',
+    });
```

Comment:
off topic.
@akshbhu it seems that we can assume that user has npm installed when using overrides. (for detector logic)

Reply:
Yup, if they have overrides/ custom working , they should have npm or yarn as package manager